compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -176,7 +176,14 @@ printTestCoverage ShowCoverage {getShowCoverage} extPkgs modules results
             ]
     pkgIdToPkgName pId = maybe pId LF.unPackageName $ join $ M.lookup pId pkgMap
     templates = [(pidM, m, t) | (pidM, m) <- modules, t <- NM.toList $ LF.moduleTemplates m]
-    choices = [(pidM, m, t, n) | (pidM, m, t) <- templates, n <- NM.names $ LF.tplChoices t]
+    choices =
+        [ (pidM, m, t, n)
+        | (pidM, m, t) <- templates
+        , n <- concat
+            [ NM.names (LF.tplChoices t)
+            , S.toList . LF.tpiInheritedChoiceNames =<< NM.toList (LF.tplImplements t)
```

Comment:
I think it might be worth preserving the information of which interface the choice was inherited from to provide more useful output on which choices you missed, e.g., something along the lines `choice C (inherited from interface I)`. Definitely doesn’t have to be in this PR though, I’d suggest to file an issue to improve this later (not urgent imho).